airbyte-integrations/connectors/source-facebook-pages/source_facebook_pages/metrics.py

```diff
@@ -245,6 +245,7 @@
         "groups",
         "image_copyrights",
         "indexed_videos",
+        # "insights_exports",    Tried accessing nonexisting field (insights_exports) on node type (Page)
```

Comment:
why is this left as comment?

Reply:
this property is commented because seems like it not supported now. I left it as commented for history.